addons/google_recaptcha/views/res_config_settings_view.xml

```diff
@@ -8,6 +8,9 @@
             <xpath expr="//setting[@id='recaptcha']" position="attributes">
                 <attribute name="help" add="If no keys are provided, no checks will be done." separator=" " />
             </xpath>
+            <xpath expr="//setting[@id='recaptcha']/field" position="replace">
+                <field name="enable_recaptcha"/>
+            </xpath>
```

Comment:
Ok, when website is not installed you still allow to enable the recaptcha app... but is there any value in having a setting to do that? Is not it automatically installed for apps who use it? To confirm, they added recaptcha for the sign up page but I don't see it as a dependency of anything other than website

Reply:
Hmmm... Isn't it better to be able to activate the `form[data-captcha]` public widgets even if the module of a form is not automatically installing reCAPTCHA module as we do for [signup forms](https://github.com/odoo/odoo/commit/37fb21911ef33885a3dae309158a359ed1a604e0)? (Which is the same thing for the `website_cf_turnstile` public widget, I guess.) :thinking: